impl/src/main/java/org/jboss/cdi/tck/tests/lookup/clientProxy/unproxyable/sealed/MyNormalScopedBean.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.cdi.tck.tests.lookup.clientProxy.unproxyable.sealed;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public sealed class MyNormalScopedBean permits MyNormalScopedSubclass {
```

Comment:
Should be called `MyNormalScoped` for consistency with other classes in this test.

Reply:
Done